opentelemetry-api/src/opentelemetry/baggage/__init__.py

```diff
@@ -18,31 +18,31 @@
 from opentelemetry.context import get_value, set_value
 from opentelemetry.context.context import Context
 
-_CORRELATION_CONTEXT_KEY = "correlation-context"
+_BAGGAGE_KEY = "baggage"
 
 
-def get_correlations(
+def get_all(
     context: typing.Optional[Context] = None,
 ) -> typing.Mapping[str, object]:
-    """Returns the name/value pairs in the CorrelationContext
+    """Returns the name/value pairs in the Baggage
 
     Args:
         context: The Context to use. If not set, uses current Context
 
     Returns:
-        Name/value pairs in the CorrelationContext
+        The name/value pairs in the Baggage
     """
-    correlations = get_value(_CORRELATION_CONTEXT_KEY, context=context)
-    if isinstance(correlations, dict):
-        return MappingProxyType(correlations.copy())
+    baggage = get_value(_BAGGAGE_KEY, context=context)
+    if isinstance(baggage, dict):
+        return MappingProxyType(baggage.copy())
     return MappingProxyType({})
 
 
-def get_correlation(
+def get_baggage(
```

Comment:
```suggestion
def get_value(
```

Reply:
I personally don't like the current names, but they are from the [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/baggage/api.md), so I assumed we should adopt them as well. Can we just choose whatever names we want for these?

Reply:
We should stick with the names outlined in the spec `get_baggage`

Reply:
That's what I thought as well. We could submit a PR to the specs if we think we have better names, but I'll mark this as resolved until then.